docs/supported-technologies.asciidoc

```diff
@@ -591,8 +591,13 @@ JBoss LogManager - 1.30.0
 |JUL - `java.util.logging`
 |All supported Java versions
 |When <<config-log-ecs-reformatting, `log_ecs_reformatting`>> is enabled, logs will be automatically reformatted into
-ECS-compatible format
-|1.31.0
+ECS-compatible format.
+
+Trace correlation is only supported when used with <<config-log-ecs-reformatting, `log_ecs_reformatting`>> as JUL does
+not provide any MDC implementation.
```

Comment:
Don't we also support log correlation when using ecs-logging-java (`co.elastic.logging.jul.EcsFormatter`) with JUL?

Reply:
Yes, good point, that's exactly for this kind of small details that I love your reviews :-).